zerver/openapi/zulip.yaml

```diff
@@ -8474,6 +8474,233 @@ paths:
                               type: string
                               description: |
                                 The text content of the status message.
+                      user_settings:
+                        type: object
+                        description: |
+                          Present if `user_settings` is present in `fetch_event_types`.
+
+                          A dictionary which contains the settings of a user.
```

Comment:
"A dictionary containing the user's personal settings."

Reply:
Done.